shortfin/python/shortfin_apps/utils.py

```diff
@@ -17,6 +17,17 @@
 from shortfin.interop.support.device_setup import get_selected_devices
 
 
+# Define disaggregation constants here for cleaner imports.
+LLM_DISAGGREGATED_PREFILL_DEVICE_IDX = 0
```

Comment:
These should be plumbed configuration details. We want to include this as flags instead of hardcoded pieces.